options_default.json

```diff
@@ -25,6 +25,7 @@
   "size_multiplier": 1.0,
   "text_color": "#FFFFFF",
   "trackerserver_authkey": "",
+  "trackerserver_twitch_id": "cc0x4xs7f13phbtgco2xeflil2pu2uy",
```

Comment:
This was removed by c07c0c856b9183a14116bdb2ae757e212bfdda42, the twitch id is no longer needed by the tracker.